packages/utils/src/axelar/bridge.ts

```diff
@@ -0,0 +1,52 @@
+import { AxelarQueryAPI, CHAINS, Environment } from "@axelar-network/axelarjs-sdk";
+
+import { domainToChainId, mainnetDomains } from "../helpers";
+
+export const chainIdToAxelarName: Map<number, string> = new Map([
```

Comment:
:(

Reply:
couldn't find a better way to get `chainId - name` mapping from axelar SDK.